src/NumField/NfAbs/NormRelation/Setup.jl

```diff
@@ -344,7 +344,7 @@ function induce_action_from_subfield(N::NormRelation, i, s, FB, cache)
   S = FB.ideals
   ZK = order(S[1])
 
-  z = SMat{ZZRingElem}[zero_matrix(SMat, FlintZZ, 0, length(S)) for i in 1:degree(field(N))]
+  z = [zero_matrix(SMat, FlintZZ, 0, length(S)) for i in 1:degree(field(N))]
```

Comment:
```suggestion
  z = sparse_matrix_type(ZZ)[zero_matrix(SMat, FlintZZ, 0, length(S)) for i in 1:degree(field(N))]
```
Lets keep helping the compiler.